src/tactic/basic.lean

```diff
@@ -101,6 +101,26 @@ meta def last : name → string
 
 end name
 
+namespace environment
+meta def decl_omap {α : Type} (e : environment) (f : declaration → option α) : list α :=
```

Comment:
I would suggest calling this `environment.filter_map`, since it's basically `decls.filter_map f`

Reply:
Fixed. I also renamed `environment.decl_map` to `environment.map`, since there's not much else to do with an `environment` that could conflict with this use of `map`.

Reply:
It seems like `environment.map` suggests that you're going to return a new `environment`. I think I still prefer `decl_map`